docs/concepts/dataframe-sql-transformation.mdx

```diff
@@ -21,4 +21,4 @@ def average_user_transaction():
    return "SELECT CustomerID, AVG(TransactionAmount) FROM {{transactions.quickstart}} GROUP BY CustomerID"
 ```
 
-In this example, `spark` represents a pre-registered Featureform object. In the *df_transform* API, explicit input setting is necessary. The decorated function should return a dataframe. Conversely, the SQL API embeds inputs using the *{{name.variant}}* or *{{name}}* syntax, depending on variant availability. The function should return a SQL-like string. This versatility enables you to harness the full potential of both SQL and dataframe transformations, tailored to your specific requirements, preferences, and infrastructure.
+In this example, `spark` represents a pre-registered Featureform object. In the **df_transform** API, explicit input setting is necessary. The decorated function should return a dataframe. Conversely, the SQL API embeds inputs using the **name.variant** or **name** syntax, depending on variant availability. The function should return a SQL-like string. This versatility enables you to harness the full potential of both SQL and dataframe transformations, tailored to your specific requirements, preferences, and infrastructure.
```

Comment:
Is there a reason you have modified the italics to bold? 

Also, we will need to leave the `{{` and `}}` because that's how users specify the table names. For example:
```
@spark.sql_transformations()
def transformation():
       return "SELECT * FROM {{table_name.variant}}" 
```
Then on our end, we parse and pull the correct table given the name and variant.

Reply:
So right now if you go to this section of the docs which is:
https://docs.featureform.com/concepts/dataframe-sql-transformation

I see a parsing error on the page and some of the content of it is missing. I used the [mdx playground](https://mdxjs.com/playground/) to zero into the cause and the culprit turned out to be this set of double braces. Do you suggest any other way that these can/should be mentioned 'cause right now these seem to be causing an issue.

Reply:
![image](https://github.com/featureform/featureform/assets/14253061/05cab273-1485-4c48-a9e4-6bc3a0b88897)

Reply:
If you rebase, you should see an update to fix the issue.

Reply:
Yup, this seems fixed now! I'll revert these changes too. 😄